Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ o2_add_dpl_workflow(hf-candidate-creator-2prong
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing ROOT::EG
                     COMPONENT_NAME Analysis)
 
-o2_add_dpl_workflow(hf-tree-creator-2prong
-                    SOURCES HFTreeCreator2Prong.cxx
+o2_add_dpl_workflow(hf-tree-creator-d0
+                    SOURCES HFTreeCreator_D0toPiKa_D0bartoKaPi.cxx
```

Comment:
- For consistency, please use the decay names as defined in `DecayType` in `HFSecondaryVertex.h`.
- If there is going to be one tree creator workflow per decay, the corresponding executable should be named accordingly.
- Underscores are not allowed in file names. ([conventions](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html))

Reply:
Thanks for the suggestions.
Actually I don't see underscores forbidden in the conventions, could you point me to the line?

Reply:
- [conventions](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=General_Naming_Rules#General_Naming_Rules)
- [from the boss](https://github.com/AliceO2Group/AliceO2/pull/5217#pullrequestreview-570988101)

Reply:
I agree I wrote this, but then I also didn't find it in the conventions for file names. @vkucera have you found it?

Reply:
> General Naming Rules
▶ Names should be meaningful; abbreviations should be avoided. They follow camel case convention. Types and variables should be nouns, while functions should be "command" verbs.

Reply:
Vit is indeed right, the camel case convention is just meaning that. I removed the underscore, I will have to do it also from other tasks (in the future). 
Many thanks to Vit!